pylintrc_tweaks

```diff
@@ -0,0 +1,8 @@
+[MASTER]
+ignore+= ,migrations, settings, wsgi.py
+
+[BASIC]
+const-rgx = (([A-Z_][A-Z0-9_]*)|(__.*__)|log|urlpatterns|logger|User)$
+
+[MESSAGES CONTROL]
+DISABLE+= ,invalid-name
```

Comment:
Looks like this is actually something that should be handled in the cookiecutter repo: https://github.com/edx/cookiecutter-django-ida/commit/d25ca33bef48344ee01786e1f1a6f35c815e8c30#diff-4eddf84cd665e6a6697c51ed3429907fR8